qa/arranger/deployments.yml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    fluxcd.io/automated: 'true'
+    fluxcd.io/tag.copy-zonefile: semver:~3.13.6
+    fluxcd.io/tag.coredns: semver:~1.8.4
+  name: arranger
+spec:
+  template:
+    spec:
+      imagePullSecrets:
+        - name: images-pull
+      volumes:
+        - name: ingress-internal-ca-certificate
+          secret:
+            secretName: clin-ingress-internal-ca-certificate
+            defaultMode: 0555
+        - name: corefile
+          configMap:
+            name: coredns-zonefiles-corefile
+            defaultMode: 0444
+        - name: zonefile
+          configMap:
+            name: qa-domain-zonefile-internal-ip
+            defaultMode: 0444
+        - name: zonefile-copy
+          emptyDir: {}
+      dnsPolicy: None
+      dnsConfig:
+        nameservers:
+          - 127.0.0.1
+          - 169.254.25.10
+        searches:
+          - search
+          - default.svc.cluster.qa
+          - svc.cluster.qa
+          - cluster.qa
+          - openstacklocal
+        options:
+          - name: ndots
+            value: "5"
+      initContainers:
+        - name: copy-zonefile
+          image: alpine:3.13.6
+          command: ["sh"]
+          args: ["-c", "cp /opt/zonefiles/* /opt/zonefiles-copy/"]
+          volumeMounts:
+            - name: zonefile
+              mountPath: /opt/zonefiles
+              readOnly: true
+            - name: zonefile-copy
+              mountPath: /opt/zonefiles-copy
+      containers:
+        - name: coredns
+          image: coredns/coredns:1.8.4
+          command: ["/coredns"]
+          args: ["-conf", "/opt/corefile/Corefile"]
+          volumeMounts:
+            - name: corefile
+              mountPath: /opt/corefile
+              readOnly: true
+            - name: zonefile-copy
+              mountPath: /opt/zonefiles
+              readOnly: true
+        - image: ferlabcrsj/clin-arranger:fba9ddc0ce38c200c0f2e3de3922fce0395a11df
+          name: arranger
+          env:
+            - name: NODE_EXTRA_CA_CERTS
+              value: /opt/ca-bundle/ca.pem
+          volumeMounts:
+            - name: ingress-internal-ca-certificate
+              mountPath: /opt/ca-bundle
+              readOnly: true
```

Comment:
You can remove the volume mount for the internal ca certificate and the node environment variable to acknowledge it. They are not needed.